deeplite_torch_zoo/api/datasets/utils.py

```
@@ -31,3 +43,78 @@ def half_precision(x):
         sampler=DS(dataset) if distributed else None,
     )
     return dataloader
+
+
+def create_ffcv_train_loader(train_dataset, num_workers, batch_size,
+                            distributed, in_memory, img_size):
+        this_device = f'cuda:{0}'
```

here you can throw an error in case FFCV not installed

```
if not FFCV_INSTALLED:
    raise RuntimeError('FFCV installation is required')
```

---

@ravishk007 you can also add a link to our installation instruction in the log message